src/domain-registry-essentials/architecture-overview.md

```diff
@@ -60,71 +60,71 @@ A domain owner or an operator may perform the following operations with their do
 
 Subdomains of any level are subjects to the same set of rules as second-level domains in CNS. Subdomains can be transferred or burned by both a direct owner and a domain zone owner \(i.e. parent domain owner\). Only a direct domain owner can set Resolver address and manage domain records.
 
-Such ownership model might not be suitable for every user. One may want to mint subdomains that are irrevocable and can be transferred only by a direct owner – this and other models are possible if a domain zone is owned by a smart contract. For more details, see [Alternative Ownership Models](architecture-overview.md#alternative-ownership-models) below.
+Such an ownership model might not be suitable for every user. One may want to mint subdomains that are irrevocable and can be transferred only by a direct owner – this and other models are possible if a domain zone is owned by a smart contract. For more details, see the [Alternative Ownership Models](architecture-overview.md#alternative-ownership-models) section below.
 
 {% hint style="info" %}
-**Note** that owner of 'crypto' top-level domain is set to a [burn address](https://etherscan.io/address/0x000000000000000000000000000000000000dEaD), that means that only direct owners can transfer or burn their second-level domains. In order to mint second-level domains we use a different mechanism, which doesn't rely on domain ownership: for more information, see [Minting Controller](architecture-overview.md#minting-controller) section.
+**Note:** If the owner of 'crypto' top-level domain is set to a [burn address](https://etherscan.io/address/0x000000000000000000000000000000000000dEaD), that means that only direct owners can transfer or burn their second-level domains. To mint second-level domains we use a different mechanism, which doesn't rely on domain ownership: for more information, see the [Minting Controller](architecture-overview.md#minting-controller) section.
 {% endhint %}
 
 ### Alternative Ownership Models
 
 Solidity smart contracts allow other models of managing subdomains to be employed. One of the ways is to use DomainZoneController smart contract, which is supported by Unstoppable Domains team.
 
-DomainZoneController suggests the model of ownership where subdomains can be managed only by their owners. Minting subdomains can be performed using whitelisted accounts, that can be added our renounced by smart contract admins.
+DomainZoneController suggests the model of ownership where subdomains can be managed only by their owners. Minting subdomains can be performed using whitelisted accounts, that can be added or renounced by smart contract admins.
```

Comment:
Good catch, that's one of my most annoying mistakes :D I always accidentally confuse "to" with "two" or "too", "our" vs "or" is a painful one too